src/Microsoft.Identity.Web.TokenAcquisition/TokenAcquisition.cs

```diff
@@ -599,8 +599,26 @@ private IConfidentialClientApplication BuildConfidentialClientApplication(Merged
 
             try
             {
+                IList<string> AddContinuousAccessEvaluationCapability(IEnumerable<string> clientCapabilities)
+                {
+                    if (clientCapabilities == null)
+                    {
+                        return new[] { Constants.CaeCapability };
+                    }
+                    else

```

Comment:
else not needed.

Reply:
Why not? 

Currently CAE is the only capability but when there will be more, we'd want to add CAE if some other ones were added already. Also currently someone can add a non-existent capability; so we'd want to add CAE also (although I don't know if STS throws an exception for invalid capabilities).

Reply:
Or are you saying just remove the else, keep the rest?